app/src/main/java/com/example/next_show/adapters/ShowAdapter.java

```diff
@@ -32,7 +33,7 @@
     private List<Show> allShows;     // list of all trending or recommended shows retrieved from Trakt
     private NavigationInterface nav;
 
-    public ShowAdapter (Context context, List<Show> shows, NavigationInterface nav){
+    public ShowAdapter (Context context, List<Show> shows, NavigationInterface nav, HashMap<String, List<String>> filters){
```

Comment:
So, the filter can change after you create the adapter right?  I think it would be better if you just update the show adapter with the final list output from the feed fragment instead of it having a reference to this filter dictionary